cli/cmd/compliance_aws.go

```diff
@@ -243,6 +245,91 @@ To show recommendation details and affected resources for a recommendation id:
 			return nil
 		},
 	}
+
+	// complianceAwsDisableReportCmd represents the disable-report sub-command inside the aws command
+	// experimental feature
+	complianceAwsDisableReportCmd = &cobra.Command{
+		Use:     "disable-report <report_type>",
+		Aliases: []string{"disable"},
+		PreRunE: func(_ *cobra.Command, args []string) error {
+			switch args[0] {
+			case "CIS":
+				args[0] = fmt.Sprintf("AWS_%s_S3", args[0])
+				return nil
+			case "AWS_CIS_S3":
+				return nil
+			default:
+				return errors.New("CIS is the only supported report type")
+			}
+		},
+		Args: cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+
+			schema, err := fetchAwsComplianceReportSchema(args[0])
+			if err != nil {
+				return err
+			}
+
+			// set state of all recommendations in this report to disabled
+			patchReq := api.NewRecommendationV1State(schema, false)
+
+			for k, v := range patchReq {
+				out := fmt.Sprintf("%s: %v \n", k, v)
+				cli.OutputHuman(out)
+			}
+
+			// _, err = cli.LwApi.Recommendations.Aws.Patch(patchReq)
+			cli.OutputHuman(fmt.Sprintf("All recommendations for report %s have been disabled\n", args[0]))
+			return nil
+		},
+	}
+
+	// complianceAwsReportStatusCmd represents the report-status sub-command inside the aws command
+	// experimental feature
+	complianceAwsReportStatusCmd = &cobra.Command{
+		Use:     "report-status <report_type>",
+		Aliases: []string{"status"},
+		PreRunE: func(_ *cobra.Command, args []string) error {
+			switch args[0] {
+			case "CIS":
+				args[0] = fmt.Sprintf("AWS_%s_S3", args[0])
+				return nil
+			case "AWS_CIS_S3":
+				return nil
+			default:
+				return errors.New("CIS is the only supported report type")
```

Comment:
Is this coming from Product? or how do we know CIS is the only supported report type?

Reply:
For AWS we are only looking to bulk disable CIS report types. This is in the ticket description written by @marcosgm 
https://lacework.atlassian.net/browse/ALLY-930